src/components/IconButton/IconButton.st.css

```diff
@@ -0,0 +1,71 @@
+:import {
+    -st-from: "wix-ui-core/index.st.css";
+    -st-named: ButtonNext;
+}
+
+:import {
+    -st-from: "../../common/formatters.st";
+    -st-named: color, fallback, font, applyOpacity;
+}
+
+:vars {
+    /*Defaults*/
+    DefaultColor: color-5;
+    DisabledColor: color-3;
+}
+
+:vars {
+    /*Overrides*/
+    IconColor: --overridable;
+}
+
+.root {
+    -st-extends: ButtonNext;
+    -st-states: disabled, skin(enum(line, full));
+    /*css goes here*/
+    background-color: transparent;
+    text-decoration: none;
+    border: none;
+    display: inline-block;
+    padding: 0;
+    margin: 0;
+    line-height: 0;
+}
+
+.root:hover {
+    opacity: 0.6;
```

Comment:
For some reason this isn't working for `line` skin.
You can see here:
https://wix-wix-ui-tpa-pr-133.surge.sh/?activeTab=Playground&path=%2F%2Fstory%2Fcomponents--iconbutton

Reply:
I can see that hover works for both skins...